grader/reject.go

```diff
@@ -125,3 +135,28 @@ func checkSubmissionLimitWindow(window *model.SubmittionLimitWindow,
 
     return nil, nil;
 }
+
+func checkLateAcknowledgment(assignment *model.Assignment, lateAcknowledgment bool) (RejectReason, error) {
+    // if the user acknowledges that they are submitting late, do not reject the submission
```

Comment:
Generally, we want comments to the be titles (Using Title Case) or complete sentences with punctuation.

Reply:
Fixed in a46adfacd6d39ffccde56fb8e6e068f677cdfa31, although this comment may change with other changes. But I will keep it in mind. Thanks!